ee/hogai/eval/conftest.py

```diff
@@ -36,12 +37,14 @@ def wrapper(*args, **kwargs):
 
 # Apply decorators to all tests in the package.
 def pytest_collection_modifyitems(items):
+    current_dir = Path(__file__).parent
     for item in items:
-        item.add_marker(
-            pytest.mark.skipif(not settings.IN_EVAL_TESTING, reason="Only runs for the assistant evaluation")
-        )
-        # Apply our custom retry decorator to the test function
-        item.obj = retry_test_only(max_retries=3)(item.obj)
+        if Path(item.fspath).is_relative_to(current_dir):
```

Comment:
Damn, yeah, I even checked pytest docs to make sure this only applies to this directory:
<img width="674" alt="Screenshot 2024-12-19 at 18 23 49" src="https://github.com/user-attachments/assets/c4a90add-40b7-40bb-8add-7712ee2ddbc3" />
and that put me at ease so much I didn't bother checking if non-eval tests will still run 🙈